src/encoded/helpers/__init__.py

```diff
@@ -0,0 +1 @@
+# package
```

Comment:
Do we need this?  It's innocuous enough...  I thought empty __init__ should be empty.

Reply:
I think this is a copy/paste thing.  I don't add it to new __init__.py files but it exists in other ones.